sklearn/linear_model/_base.py

```diff
@@ -560,8 +560,15 @@ def fit(self, X, y, sample_weight=None):
         y : array-like of shape (n_samples,) or (n_samples, n_targets)
             Target values. Will be cast to X's dtype if necessary.
 
-        sample_weight : array-like of shape (n_samples,), default=None
-            Individual weights for each sample.
+        sample_weight : array-like of shape (n_samples,), int, float or None (default)
```

Comment:
no need to have float and int. float covers both. In the few places where we already mention float as valid type, it comes first. Let's keep the same formating for consistency.
```suggestion
        sample_weight : float or array-like of shape (n_samples,), default=None
```